modules/ROOT/pages/con_tasks-syntax.adoc

```diff
@@ -0,0 +1,74 @@
+[]
+= Tasks Syntax
+
+To define a configuration file for a `Task` resource, specify the
+following fields:
+
+* Required:
+  * `apiVersion` - Specifies the API version.
+  * `kind` - Specify the `Task` resource object
+  * `metadata` - Specifies data to uniquely identify the
+    `Task` resource object, for example a `name`.
+  * `spec` - Specifies the configuration information for
+    your `Task` resource object. `Task` steps must be defined through either of
+    the following fields:
+  ** link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#steps[`steps`] - Specifies one or more container images that you want
+      to run in your `Task`.
+
+* Optional:
+  * link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#inputs[`inputs`] - Specifies parameters and `PipelineResources` needed by your `Task`. A `Task` can declare the inputs it needs, which can be either or both of:
+
+** link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#parameters[`parameters`]
+** link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#input-resources[input resources] 
+ 
+  * link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#steps[`outputs`] - Specifies `PipelineResources`
+    created by your `Task`
+  * link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#volumes[`volumes`] - Specifies one or more volumes that you want to make
+    available to your `Task`'s steps.
+  * link:https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md#step-template[`stepTemplate`] - Specifies a `Container` step
+    definition to use as the basis for all steps within your `Task`.
+
+
+The following example shows a YAML file where the configuration fields are completed:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: example-task-name
+spec:
+  inputs:
+    resources:
+      - name: workspace
+        type: git
+    params:
+      - name: pathToDockerFile
+        type: string
+        description: The path to the dockerfile to build
+        default: /workspace/workspace/Dockerfile
+  outputs:
+    resources:
+      - name: builtImage
+        type: image
+  steps:
+    - name: ubuntu-example
+      image: ubuntu
+      args: ["ubuntu-build-example", "SECRETS-example.md"]
+    - image: gcr.io/example-builders/build-example
+      command: ["echo"]
+      args: ["$(inputs.params.pathToDockerFile)"]
+    - name: dockerfile-pushexample
+      image: gcr.io/example-builders/push-example
+      args: ["push", "$(outputs.resources.builtImage.url)"]
+      volumeMounts:
+        - name: docker-socket-example
+          mountPath: /var/run/docker.sock
+  volumes:
+    - name: example-volume
+      emptyDir: {}
+----
+
+== Additional resources
+
+For more information see the documentation on link:https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields[Required Fields]
```

Comment:
For more information see the documentation on link:https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields[Required Fields]-->
For more information see link:https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields[Required Fields]